examples/todo/src/__tests__/acceptance/todo.acceptance.ts

```diff
@@ -87,7 +87,8 @@ describe('TodoApplication', () => {
   });
 
   it('creates an address-based reminder', async function() {
-    if (!available) return;
+    // eslint-disable-next-line no-invalid-this
+    if (!available) return this.skip();
```

Comment:
While we won't be notified, would it be worth it to do something like `console.warn('service down, skipping test')`?

Reply:
IDK, to be honest 🤷‍♂